plugins/modules/firewall.py

```diff
@@ -83,13 +82,12 @@
         ports: 80
         destinations:
           - addresses:
-            - "0.0.0.0/0"
-            - "::/0"
+              - "0.0.0.0/0"
+              - "::/0"
```

Comment:
@mamercad If you notice this spacing is also 4-point spacing when the rest of the yaml is 2-point spacing, why ansible-lint was failing on this I cannot say, but I decided it was easier to acquiesce then question further.